src/controllers/studyMate.controller.js

```diff
@@ -48,6 +48,23 @@ const offer = async (req, res, next) => {
   );
 };
 
+const posts = async (req, res, next) => {
+  const page = req.query.page || 1; // 현재 페이지 번호 (기본값 : 1)
+  const limit = 10; // 페이지당 글 개수
+  const offset = limit * (page - 1);
+  const state = req.query.state || "최신순"; // 정렬 순서 결정
+
+  let focus = "";
+  if (state === "최신순") {
+    focus = "DESC";
+  } else {
+    focus = "ASC";
+  }
```

Comment:
```suggestion
  const focus = req.query.state === "최신순" ? "DESC" : "ASC";
```
다 잘했는데 이거 한줄로 줄일 수 있어서 이거만 한번 봐주면 될것 같아